styleguide.config.js

```diff
@@ -103,16 +103,18 @@ module.exports = {
       sidebarBackground: 'white',
     },
     fontFamily: {
-      base: '"Open Sans", system-ui, sans-serif',
+      base: '"Open Sans", system-ui, sans-serif', // --mds-type-font-family
```

Comment:
Just curious: is there no way to grab the value of a CSS variable and include it in a javascript object as a string?

Reply:
AFAIK, no. That would require a Webpack loader to parse the file. But, we're just booting up Styleguidist (a native Node process, without any Webpack) so we don't have any of that.